### DISCUSSION TITLE
How to add a LORA to this workflow?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [comfyanonymous/ComfyUI#1044] [Q&A] [answered]
![Screenshot_3](https://github.com/comfyanonymous/ComfyUI/assets/13344308/a554d040-c413-4e22-8c0b-857d0dda01ca)

**Comment by fabiobtex** [comment, marked as answer]
Add node, loaders, load LoRa, connect in the base model.
https://comfyanonymous.github.io/ComfyUI_examples/lora/

**Reply by ZeroCool22** [reply]
Thx.
